### PR TITLE
Update configuration page

### PR DESCRIPTION
**Pull request by itsgareth**
updates the configuration page to match the best practices.

**Before & After:**
<div>
Screenshot 2023-10-09 at 12 52 04

Screenshot 2023-10-09 at 12 50 54
<div>

### CODE REVIEW

File: src/components/ConfigEditor.test.tsx [hunk 1]
```diff
@@ -4,44 +4,6 @@ import { render, screen } from '@testing-library/react';
 import { ConfigEditor } from './ConfigEditor';
 
 describe('ConfigEditor', () => {
-  it('should support old authType property', () => {
```
Comment (zoltanbedi) [inline src/components/ConfigEditor.test.tsx:7]: Why was these tests removed?

Comment (gwdawson) [reply src/components/ConfigEditor.test.tsx]: @zoltanbedi thats my bad, i assumed these were related to to removal of `// Backward compatibility check. SecretInput was added in 8.5`.

going back and checking this isnt the reason they failed, i updated these to pass again.

File: jest.config.js [hunk 2]
```diff
@@ -5,5 +5,7 @@ module.exports = {
   // Jest configuration provided by @grafana/create-plugin
   ...require('./.config/jest.config'),
   // Inform jest to only transform specific node_module packages.
-  transformIgnorePatterns: [nodeModulesToTransform([...grafanaESModules, 'rxjs'])],
+  transformIgnorePatterns: [
```
Comment (zoltanbedi) [inline jest.config.js:8]: This shouldn't be necessary. I'll fix this in a separate PR.

**Review by zoltanbedi — APPROVED** [review]
LGTM, we need to bump the grafana dependency now to 9 in the plugin.json

**Pull request merged**